themes/mongodb/src/css/stitch/stitch.css

```diff
@@ -0,0 +1,1584 @@
+/*
```

Comment:
This makes me nervous.

What's actually different about this, compared to the source this was copied from?

Reply:
This is generally the same as `mongodb-docs.css`, with the following changes:
1) Moved styles that affect the left-hand Sphinx sidebar from the main css file to `stitch-sidebar.css`
2) Removed unused styles and styles that are no longer applicable to the new TOC sidebar.
3) Added new styles in `stitch-sidebar.css`

afaict it's necessary to split out a separate `stitch.css` file for this because (1) the new styles conflict with the old (2) imported styles are always overwritten unless we use `!important`, which is gross and hard to keep track of.